csuntechlab/iSTART
# STR-141 -- FE: Create Multi-Question text-input (1-100) Quiz Slide + Answer Slide template

# Description
  To create the Multi-Question text-input (1-100) Quiz Slide + Answer Slide template

# Testing
'What does the reviewer need to do to test your PR?'

### Preparations for testing
1. Run `yarn watch`.
2. Update **`Module.vue`** to reflect the snippet below for testing the template.
```
<template>
  <div ref="moduleContainer" :class="checkWindowWidth">
    <Navbar></Navbar>
    <module-header :size_of_container="sizeOfContainer"></module-header>
    <quiz-input-template v-if="slideNumber !== 2 && slide_type=== 'quizInput'"></quiz-input-template>
    <pie-chart-template v-if="slideNumber === 2"></pie-chart-template>
    <module-footer></module-footer>
  </div>
</template>
```

### Testing Functionality
1. Login as `april.feldman` or `cyndee.bellamy`.
2. In the **Dashboard**, visit the **Alcohol** module.
3. Attempt to break form validation and ensure all requirements are met to be able to continue.

>- Numeric Values
>- Range of ***0-100***
>- Positive Integers
>- All inputs filled to continue
>- All inputs valid
>- Invalid inputs remove continue even if continue was allowed previously

4. Enter valid inputs and click continue.
5. Check pie charts properly present data side by side from what the **user** said against what **they** said.
 5. Make sure the code is fire; pour one for Josh who did 99% of the code and didn't get to see it pushed up on dev.

# Installation
Run ***`yarn install`***
